src/default_panels/StyleTracesPanel.js

```diff
@@ -653,11 +659,38 @@ const StyleTracesPanel = (props, {localize: _}) => (
       )}
       mode="trace"
     >
-      <DataSelector label={_('Text')} attr="text" />
       <TextPosition label={_('Text Position')} attr="textposition" />
+      <HoverTemplateSwitch attr="texttemplate" label={_('Mode')} />
+      <TextInfo attr="textinfo" label={_('Show')} />
+      <HoverTemplateText attr="texttemplate" label={_('Template')} />
```

Comment:
`attr="texttemplate"` - we have it twice here?

Reply:
it's a too-clever way of building the "switch" control

Reply:
one is the switch and the other is the template text